File: app/src/main/java/com/veganbeauty/app/features/myskin/BookingSuccessFragment.java
package com.veganbeauty.app.features.myskin;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.FragmentManager;

import com.veganbeauty.app.R;
import com.veganbeauty.app.core.base.RootieFragment;
import com.veganbeauty.app.features.home.HomeFragment;

public class BookingSuccessFragment extends RootieFragment {

    private String bookingId;
    private String storeName;
    private String dateTime;
    private String specialist;
    private String serviceName;

    public static BookingSuccessFragment newInstance(String bookingId, String storeName, String dateTime, String specialist, String serviceName) {
        Bundle args = new Bundle();
        args.putString("BOOKING_ID", bookingId);
        args.putString("STORE_NAME", storeName);
        args.putString("DATE_TIME", dateTime);
        args.putString("SPECIALIST", specialist);
        args.putString("SERVICE_NAME", serviceName);
        BookingSuccessFragment fragment = new BookingSuccessFragment();
        fragment.setArguments(args);
        return fragment;
    }

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        return inflater.inflate(R.layout.skin_fragment_booking_success, container, false);
    }

    @Override
    public void setupUI(@NonNull View view) {
        if (getArguments() != null) {
            bookingId = getArguments().getString("BOOKING_ID", "");
            storeName = getArguments().getString("STORE_NAME", "Rootie Gò Vấp");
            dateTime = getArguments().getString("DATE_TIME", "");
            specialist = getArguments().getString("SPECIALIST", "Nguyễn Khánh Xuân");
            serviceName = getArguments().getString("SERVICE_NAME", "");
        } else {
            bookingId = "";
            storeName = "Rootie Gò Vấp";
            dateTime = "";
            specialist = "Nguyễn Khánh Xuân";
            serviceName = "";
        }

        TextView tvBookingId = view.findViewById(R.id.info_booking_id);
        TextView tvStoreName = view.findViewById(R.id.info_store_name);
        TextView tvDateTime = view.findViewById(R.id.info_date_time);
        TextView tvSpecialist = view.findViewById(R.id.info_specialist);
        TextView tvService = view.findViewById(R.id.info_service);
        
        ImageView btnBack = view.findViewById(R.id.btn_back);
        TextView btnHome = view.findViewById(R.id.btn_home);

        tvBookingId.setText(bookingId);
        tvStoreName.setText(storeName);
        tvDateTime.setText(dateTime);
        tvSpecialist.setText(specialist);
        tvService.setText(serviceName);

        btnBack.setOnClickListener(v -> getParentFragmentManager().popBackStack());

        btnHome.setOnClickListener(v -> {
            FragmentManager fm = getParentFragmentManager();
            for (int i = 0; i < fm.getBackStackEntryCount(); i++) {
                fm.popBackStack();
            }
            fm.beginTransaction()
                .replace(R.id.main_container, new HomeFragment())
                .commit();
        });
    }

    @Override
    protected void observeViewModel() {
        // Not used
    }
}

File: app/src/main/java/com/veganbeauty/app/features/myskin/BookingSuccessFragment.kt
package com.veganbeauty.app.features.myskin

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.ImageView
import android.widget.TextView
import com.veganbeauty.app.R
import com.veganbeauty.app.core.base.RootieFragment
import com.veganbeauty.app.features.home.HomeFragment

class BookingSuccessFragment : RootieFragment() {

    private lateinit var storeName: String
    private lateinit var dateTime: String
    private lateinit var specialist: String
    private lateinit var serviceName: String

    companion object {
        fun newInstance(storeName: String, dateTime: String, specialist: String, serviceName: String): BookingSuccessFragment {
            val args = Bundle()
            args.putString("STORE_NAME", storeName)
            args.putString("DATE_TIME", dateTime)
            args.putString("SPECIALIST", specialist)
            args.putString("SERVICE_NAME", serviceName)
            val fragment = BookingSuccessFragment()
            fragment.arguments = args
            return fragment
        }
    }

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        return inflater.inflate(R.layout.skin_fragment_booking_success, container, false)
    }

    override fun setupUI(view: View) {
        storeName = arguments?.getString("STORE_NAME") ?: "Rootie Gò Vấp"
        dateTime = arguments?.getString("DATE_TIME") ?: ""
        specialist = arguments?.getString("SPECIALIST") ?: "Nguyễn Khánh Xuân"
        serviceName = arguments?.getString("SERVICE_NAME") ?: ""

        val tvStoreName: TextView = view.findViewById(R.id.info_store_name)
        val tvDateTime: TextView = view.findViewById(R.id.info_date_time)
        val tvSpecialist: TextView = view.findViewById(R.id.info_specialist)
        val tvService: TextView = view.findViewById(R.id.info_service)
        
        val btnBack: ImageView = view.findViewById(R.id.btn_back)
        val btnHome: TextView = view.findViewById(R.id.btn_home)

        tvStoreName.text = storeName
        tvDateTime.text = dateTime
        tvSpecialist.text = specialist
        tvService.text = serviceName

        btnBack.setOnClickListener { 
            // Về màn hình trước đó
            parentFragmentManager.popBackStack() 
        }

        btnHome.setOnClickListener {
            // Về trang chủ của app
            // Cần pop toàn bộ backstack và chuyển về HomeFragment
            val fm = parentFragmentManager
            for (i in 0 until fm.backStackEntryCount) {
                fm.popBackStack()
            }
            fm.beginTransaction()
                .replace(R.id.main_container, HomeFragment())
                .commit()
        }
    }

    override fun observeViewModel() {
        // Not used
    }
}
